release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1263,11 +1263,16 @@ In the following tables, features are marked with the following statuses:
 |General Availability
 |General Availability
 
-|Customized `br-ex` bridge needed by OVN-Kuberenetes to use NMState
+|OVN-Kuberenetes customized `br-ex` bridge on bare metal
 |General Availability
 |General Availability
 |General Availability
 
+|OVN-Kuberenetes customized `br-ex` bridge on {vmw-short} and {op-system}
```

Comment:
It's not RHCOS, it's OpenStack that should be the other platform here.

Reply:
Ack. Updated!